src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -643,6 +981,7 @@ public void updateSubscribers()
         isBuildingsDirty = false;
         permissions.clearDirty();
 
+        // TODO: look into potentially keeping lists as well
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)